include/xtensor/xexpression.hpp

```diff
@@ -391,52 +404,47 @@ namespace xt
      ********************************/
 
     template <class... E>
-    struct xoptional_comparable : xtl::conjunction<xtl::disjunction<is_xtensor_expression<E>,
-                                                                    is_xoptional_expression<E>
-                                                                   >...
-                                                  >
+    struct xoptional_comparable
+        : xtl::conjunction<xtl::disjunction<is_xtensor_expression<E>, is_xoptional_expression<E>>...>
     {
     };
 
-#define XTENSOR_FORWARD_CONST_METHOD(name)                                      \
-    auto name() const                                                           \
-        -> decltype(std::declval<xtl::constify_t<E>>().name())                  \
-    {                                                                           \
-        return m_ptr->name();                                                   \
+#define XTENSOR_FORWARD_CONST_METHOD(name)                                 \
+    auto name() const->decltype(std::declval<xtl::constify_t<E>>().name()) \
+    {                                                                      \
+        return m_ptr->name();                                              \
     }
 
-#define XTENSOR_FORWARD_METHOD(name)                                            \
-    auto name() -> decltype(std::declval<E>().name())                           \
-    {                                                                           \
-        return m_ptr->name();                                                   \
+#define XTENSOR_FORWARD_METHOD(name)                \
+    auto name()->decltype(std::declval<E>().name()) \
```

Comment:
The formatting of `->` is incoherent with the formatting below. What is going on here?